easybuild/easyconfigs/s/setuptools/setuptools-23.1.0-GCC-5.4.0-2.26-Python-2.7.12.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "PythonPackage"
+
+name = "setuptools"
+version = "23.1.0"
+
+homepage = "https://pypi.python.org/pypi/setuptools/"
+description = """Download, build, install, upgrade, and uninstall Python packages -- easily!"""
+
+toolchain = {'name': 'GCC', 'version': '5.4.0-2.26'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['4e269d36ba2313e6236f384b36eb97b3433cf99a16b94c74cca7eee2b311f2be']
+
+python = "Python"
+pythonversion = "2.7.12"
+pythonsuffix = "-bare"
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [(python, pythonversion, pythonsuffix)]
```

Comment:
please clean this up, drop the `python`, `pythonversion` local variables and use the `%(pyver)s` and `%(pyshortver)s` templates instead?

Also, use `versionsuffix = '-Python-%(pyver)s`?